docs/getting-started/quickstart.md

```diff
@@ -98,4 +83,14 @@ docker run -it --rm \
 	n8n start --tunnel
 ```
 
-In case you run into issues, check out the [troubleshooting](../reference/troubleshooting.md) page or ask for help in the community [forum](https://community.n8n.io/).
+In case you run into issues with the installation, check out the [troubleshooting page](../reference/troubleshooting.md) or ask for help in the [community forum](https://community.n8n.io/).
+
+# Run n8n remotely
```

Comment:
I wonder if mentioning 'remotely' would confuse people? Would it be clearer if we ask say, 'Run n8n with n8n.cloud'.